packages/react-native/ReactCommon/react/renderer/imagemanager/platform/ios/react/renderer/imagemanager/RCTImagePrimitivesConversions.h

```diff
@@ -117,5 +154,9 @@ inline static NSURLRequest *NSURLRequestFromImageSource(const facebook::react::I
     }
   }
 
+  request.HTTPBody = body;
+  request.HTTPMethod = method;
+  request.cachePolicy = cachePolicy;
+
   return [request copy];
```

Comment:
We believe we can just return the `request` here and save the copy.
The request is a local object of the method, so it will not be referenced by anything else. It is safe to return the reference.